### ISSUE TITLE
About the pretrained model

### ISSUE THREAD

**Issue by horizon0408** [lelimite4444/BridgeDepthFlow#3]
Hi!

Appreciate your code and work. I have small questions with the pre-trained model you provided.
Is the model monodepth_ver_a/b/c correspond to the full-1/2/3 in the Table 1 and Table 2 in your paper? I run your test and evaluation script with monodepth_ver_a on KITTI2015 and get the results like:
   abs_rel,     sq_rel,        rms,    log_rms,     d1_all,         a1,         a2,         a3
    0.0648,     0.8067,      4.186,      0.157,      8.623,      0.946,      0.979,      0.990
which is slightly different from Table 1. So I'm not sure if the model is the corresponding one or I missed some details for evaluation.

**Comment by lelimite4444** [comment]
Hi.
Sorry for confusing, because I lost my model weights which corresponds to the results on my paper when I was preparing the code.
I trained it again, so there’s slightly difference. The version a/b/c correspond to full-1,2,3 exactly.

**Comment by zmlshiwo** [comment]
Hi, @lelimite4444 
Last time, I tried to reproduce the results of full-2. First, I trained the monodepth without 2warp loss about 40 epochs. Then, I used this model as pre-trianed model and further trained this pre-trianed model with 2warp loss about 80epochs. I obtain the following results.
abs_rel, sq_rel, rms, log_rms, d1_all, a1, a2, a3
0.0660,     0.8078,      4.240,      0.152,      8.901,      0.944,      0.978,      0.989
These results are slightly worse than your results in paper. Did I ignore any details?
In stage of two, we only load the pre-trianed model as initialization, and the learning rate is similar to the stage one.
In addition, I saw your pre-trained model using PWC-Net, which was trained with 40epochs. Does it mean the PWC-Net is first trained with 40 epochs (no 2warp) as initialization and then further trained 40 epochs with 2warp?

**Comment by wanghao14** [comment]
@horizon0408 @zmlshiwo Hi, may I ask you a question? Because I emailed the author and he didn't reply. When the author make the occlusion estimation for optical flow in the [#128](https://github.com/lelimite4444/BridgeDepthFlow/blob/master/utils/utils.py#L128) of utils/utils.py, which corresponds to the formula (6) in the paper, the value of mag_sq is not consistent with the formula. Is there any typo here, or he did it for the convenience of calculation?

**Comment by amarzullo24** [comment]
Hi!
I noticed that the pretrained model in the [drive folder](https://drive.google.com/drive/folders/1l6HLF2Dq1ObsTjdJaAPKK6NOAZDOI0Rs) is not available anymore. Can someone share it again?

Thanks!
